gateway/crates/federated-server/src/server/engine.rs

```diff
@@ -36,6 +37,7 @@ pub(super) async fn post(
     headers: HeaderMap,
     Json(request): Json<engine::BatchRequest>,
 ) -> impl IntoResponse {
+    state.request_counter().add(1, &[]);
```

Comment:
I'm guessing the `&[]` is for metric tags? would be nice to know the http method and response status.

Reply:
It's just a dummy request counter to have something to test against in the integration test^^ Will be improved later

Reply:
and yeah those are the attributes